JokeRandomizer/JokeRandomizer/CustomButtons/CustomButton.swift

```diff
@@ -21,11 +21,12 @@ class CustomButton: UIButton {
     
     func setupButton() {
         setShadow()
-        
+        shake()
+        let image = UIImage(systemName: "arrow.clockwise.circle.fill")
```

Comment:
don't use hard coded text directly. use a constants instead. You may create a private enum Constants for values distinct to the class it is being used. For common values being used globally, make a global Constants. This applies to various data type like String, Int, Double, etc...

Reply:
make a function to call in the update